### PR TITLE
Feature/restart 1

### PR DESCRIPTION
**Pull request by jschaul**
cassandra, elasticsearch

### CODE REVIEW

File: source/how-to/administrate/cassandra.rst [hunk 1]
```diff
@@ -27,9 +27,19 @@ How to inspect tables and data manually
...
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For maintenance you may need to restart the cluster.
+
+On each server one by one:
 
-TODO
+1. check your cluster is healthy: ``nodetool status``
+2. ``nodetool drain`` (to stop accepting writes and flush data to disk)
+3. ``systemctl stop cassandra`` (to stop the process)
```
Comment (tiago-loureiro) [inline source/how-to/administrate/cassandra.rst]: Maybe we could rephrase this into: "Stop the cassandra daemon" - on systemd based systems (or if you used our ansible plyabooks), it should be `systemctl stop cassandra`.

File: source/how-to/administrate/cassandra.rst [hunk 2]
```diff
@@ -27,9 +27,19 @@ How to inspect tables and data manually
...
+4. do any operation you need, if any
+5. ``systemctl start cassandra``
```
Comment (tiago-loureiro) [inline source/how-to/administrate/cassandra.rst]: Similar to the above?

File: source/how-to/administrate/elasticsearch.rst [hunk 3]
```diff
@@ -3,12 +3,40 @@ Elasticsearch
...
+2. stop shard allocation:
+
+.. code:: sh
+
+    ES_IP=<the-ip-of-the-elasticsearch-node-to-stop>
+    curl -sSf -XPUT http://localhost:9200/_cluster/settings -d "{ \"transient\" : {\"cluster.routing.allocation.exclude._ip\": \"$ES_IP\" }}"
+
+3. ``systemctl stop elasticsearch`` (to stop the process)
+4. do any operation you need, if any
+5. ``systemctl start elasticsearch``
```
Comment (tiago-loureiro) [inline source/how-to/administrate/elasticsearch.rst]: Same idea as the above for systemd based systems?

File: source/how-to/administrate/cassandra.rst (hunk 1 above)
Comment (jschaul) [reply source/how-to/administrate/cassandra.rst]: The intro at the top of the page reads:

```
This section is about **how to perform a specific task**. If you want to **understand how a certain component works, please see** :ref:`understand`

The rest of the page assumes you installed using the ansible playbooks from `wire-server-deploy <https://github.com/wireapp/wire-server-deploy/tree/master/ansible>`__
```

We can't cover all possible ways people have installed cassandra, so these instructions only work on systemd based systems, but the note at the top should make that clear.
I'll reformulate slightly though to make it clearer what that command does for someone unfamiliar with systemd.

**Pull request merged**